iop/fs/bdm/src/part_driver.c

```diff
@@ -234,6 +254,7 @@ void part_init()
     }
 
     g_part_fs.priv          = NULL;
+    // TODO: switch from hardcoded string to variable, at least "MBR" and "Removable"
     g_part_fs.name          = "MBR";
```

Comment:
No, this is the "MBR" file system driver. There is no need for a "Removable" driver, becouse it's already the entire block device.